src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/ObjectionStatus.java

```diff
@@ -6,6 +6,9 @@
     INELIGIBLE_NO_DISSOLUTION_ACTION,
     OPEN,
     PROCESSED,
+    PROCESSING,
```

Comment:
The status in the story are ERROR_... to be set when the exception is caught. So when chips fails, the sendObjectionToChips method will set the status to ERROR_CHIPS, likewise for sendInternalEmail and sendExternalEmail. Keep 'Processing' though, that might be useful.